docs/usage/9-getting-deterministic-logical-factories.md

```diff
@@ -0,0 +1,140 @@
+# Getting Deterministic Logical Values
+
+In this following example we create a factory for a `Person` model
+
+```py
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+```
+
+If we use the factory class:
+
+```py
+from pydantic_factories import ModelFactory
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+
+
+class PersonFactory(ModelFactory):
+    __model__ = Person
+```
+
+We can get values like this:
+
+```json
+{
+  "values": [
+    {
+      "name": "GMPXMVbyXHUfymCDaloV",
+      "age": 6380
+    },
+    {
+      "name": "iPhGgaTklZTpXCJkWJYU",
+      "age": 4895
+    },
+    {
+      "name": "bvWnrQEEKdspKjyjnWXs",
+      "age": 6623
+    }
+  ]
+}
+```
+
+So we have 2 problems with that.<br>
+The next run will lead to completely different values.<br>
+The values are not logical (age=6623)
+
+## Using Seed & Choice
+
+First, we need to make the random part deterministic. We can achieve that like this:
+
+```py
```

Comment:
Everything from here till line 90 is very redundant, I think its best to remove it & the example code snippet shared towards EOF is more than enough.